content/guides/nextjs/adding-tina/adding-tina-provider.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Adding the Tina Provider
+---
+
+We need to wrap every page in the `<TinaProvider>` component. This component will provide the CMS to all of our pages, allowing us to create an editor for our content. We can do this in Next.js by [creating a custom _App_ component](https://nextjs.org/docs#custom-app). Next will then use our custom app component to render the page.
+
+Our blog starter already has this file created. Open up `pages/_app.js` and you should see something like this:
+
+```jsx
+import '../styles/index.css'
+
+export default function MyApp({ Component, pageProps }) {
+  return <Component {...pageProps} />
+}
+```
+
+## Installation
+
+From here, install `tinacms`. We also need to install `styled-components`, as it's a peer dependency:
+
+```bash
+yarn add tinacms styled-components
```

Comment:
I also had to install `moment` before I was able to run the development server.

Reply:
Good catch! I believe next week's Tina release will have moment configured as a dependency, so that won't be necessary by the time this is published